Waiting for unfinished jobs

```cmd
Linux version 5.7.8-1.el7.elrepo.x86_64 (mockbuild@Build64R7) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-39) (GCC), GNU ld version 2.27-43.base.el7_8.1)

make[2]: Leaving directory `/opt/netdata/netdata-v1.25.0/aclk'
Making all in spawn
make[2]: Entering directory `/opt/netdata/netdata-v1.25.0/spawn'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/opt/netdata/netdata-v1.25.0/spawn'
make[2]: Entering directory `/opt/netdata/netdata-v1.25.0'
  CC       libnetdata/json/jsmn.o
  CC       web/server/web_client.o
  CC       web/server/web_server.o
  CC       web/server/web_client_cache.o
  CC       web/server/static/static-threaded.o
web/server/web_client.c: In function 'http_header_parse':
web/server/web_client.c:819:14: error: 'struct web_client' has no member named 'ssl'
             w->ssl.flags |= NETDATA_SSL_PROXY_HTTPS;
              ^
web/server/web_client.c:819:29: error: 'NETDATA_SSL_PROXY_HTTPS' undeclared (first use in this function)
             w->ssl.flags |= NETDATA_SSL_PROXY_HTTPS;
                             ^
web/server/web_client.c:819:29: note: each undeclared identifier is reported only once for each function it appears in
web/server/web_client.c: In function 'web_client_switch_host':
web/server/web_client.c:1355:27: error: 'struct web_client' has no member named 'ssl'
             protocol = ((w->ssl.conn && !w->ssl.flags) || w->ssl.flags & NETDATA_SSL_PROXY_HTTPS) ? "https" : "http";
                           ^
web/server/web_client.c:1355:43: error: 'struct web_client' has no member named 'ssl'
             protocol = ((w->ssl.conn && !w->ssl.flags) || w->ssl.flags & NETDATA_SSL_PROXY_HTTPS) ? "https" : "http";
                                           ^
web/server/web_client.c:1355:60: error: 'struct web_client' has no member named 'ssl'
             protocol = ((w->ssl.conn && !w->ssl.flags) || w->ssl.flags & NETDATA_SSL_PROXY_HTTPS) ? "https" : "http";
                                                            ^
web/server/web_client.c:1355:74: error: 'NETDATA_SSL_PROXY_HTTPS' undeclared (first use in this function)
             protocol = ((w->ssl.conn && !w->ssl.flags) || w->ssl.flags & NETDATA_SSL_PROXY_HTTPS) ? "https" : "http";
                                                                          ^
make[2]: *** [web/server/web_client.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory `/opt/netdata/netdata-v1.25.0'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/opt/netdata/netdata-v1.25.0'
make: *** [all] Error 2
 FAILED  
```

--- a/web/server/web_client.c
+++ b/web/server/web_client.c
@@ -814,10 +814,12 @@ static inline char *http_header_parse(struct web_client *w, char *s, int parse_u
         }
     }
 #endif /* NETDATA_WITH_ZLIB */
+#ifdef ENABLE_HTTPS
     else if(hash == hash_forwarded_proto && !strcasecmp(s, "X-Forwarded-Proto")) {
         if(strcasestr(v, "https"))
             w->ssl.flags |= NETDATA_SSL_PROXY_HTTPS;
     }
+#endif
     else if(hash == hash_forwarded_host && !strcasecmp(s, "X-Forwarded-Host")){
         strncpyz(w->forwarded_host, v, ((size_t)(ve - v) < sizeof(w->server_host)-1 ? (size_t)(ve - v) : sizeof(w->server_host)-1));
     }
@@ -1352,7 +1354,11 @@ static inline int web_client_switch_host(RRDHOST *host, struct web_client *w, ch
         if(!url) { //no delim found
             debug(D_WEB_CLIENT, "%llu: URL doesn't end with / generating redirect.", w->id);
             char *protocol, *url_host;
+#ifdef ENABLE_HTTPS
             protocol = ((w->ssl.conn && !w->ssl.flags) || w->ssl.flags & NETDATA_SSL_PROXY_HTTPS) ? "https" : "http";
+#else
+            protocol = "http";
+#endif
             url_host = (!w->forwarded_host[0])?w->server_host:w->forwarded_host;
             buffer_sprintf(w->response.header, "Location: %s://%s%s/\r\n", protocol, url_host, w->last_url);
             buffer_strcat(w->response.data, "Permanent redirect");

